darshan-runtime/doc/darshan-runtime.rst

```diff
@@ -83,10 +83,8 @@ Compilation
 
     tar -xvzf darshan-<version-number>.tar.gz
     cd darshan-<version-number>/
-    ./prepare.sh
```

Comment:
Maybe somewhere we should have a note for people that clone from git that they need to run `./prepare.sh`?  These instructions are correct for the release tarballs.